### DISCUSSION TITLE
what ER region is

### DISCUSSION THREAD

**Discussion by tripleenthusiast** [Mr-MIBonk/M.I.B._More-Incredible-Bash#180] [Q&A] [answered]
I got MHI2Q_**ER**_AUG22_P5092 (1329) version.

Simple question. **What means ER?** I know from wiki its region but which region? What a shortcut is that?

**Comment by harman-f** [comment]
EU/ROW —> ER

**Reply by tripleenthusiast** [reply]
idk I am dumb what ROW means?

**Reply by harman-f** [reply]
Rest of world

**Reply by tripleenthusiast** [reply]
rest of world? but what rest?

**Reply by tripleenthusiast** [reply]
ok, is it defined what rest of world?

**Reply by harman-f** [reply, marked as answer]
Well, there is country specific FW for:
US, JP, KR, CN, TW
And for everyone else there is ER
